libsolidity/formal/SMTEncoder.h

```diff
@@ -295,6 +295,10 @@ class SMTEncoder: public ASTConstVisitor
 	/// otherwise nullptr.
 	MemberAccess const* isEmptyPush(Expression const& _expr) const;
 
+	/// @returns true if the given identifier is a contract which is known and trusted.
+	/// This means we don't have to abstract away effects of external function calls to this contract.
+	static bool isTrustedContract(Identifier const* _identifier);
```

Comment:
I think this should take an `Expression` instead

Reply:
Changed.